core/tauri-runtime-wry/src/lib.rs

```diff
@@ -1409,6 +1423,19 @@ fn handle_event_loop(
   }
 }
 
+fn center_window(window: &Window) -> Result<()> {
+  if let Some(monitor) = window.current_monitor() {
```

Comment:
I think this will panic or at the very least won't work, the monitor module isn't implemented for gtk yet.

Reply:
yeah it'll panic :/ the APIs are there if people wanna use on mac/windows

Reply:
Then we should mention it somewhere so people won't open issues about it.